microceph/common/fileUtils.go

```diff
@@ -0,0 +1,43 @@
+package common
```

Comment:
Style nit: per convention, package and file names should be single, lowercase words. Suggest fileUtils.go -> fileutils.go